website/docs/docs/use-dbt-semantic-layer/dbt-sl.md

```diff
@@ -38,6 +38,10 @@ product="dbt Semantic Layer"
 plan="dbt Starter or Enterprise-tier"
 />
 
+:::info Not yet supported in Fusion
```

Comment:
Branding guidelines stipulate the first mention of Fusion on any page must say "the dbt Fusion engine." Doesn't look like it's mentioned elsewhere here, so we'll have to address it with the callout
```suggestion
:::info Not yet supported in the dbt Fusion engine
```